k8s/base-config.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: innsecure
+  labels:
+    name: innsecure
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: innsecure-config
+  namespace: innsecure
+  labels:
+    app: innsecure
+data:
+  POSTGRES_DB: innsecure
+  POSTGRES_USER: root
+  POSTGRES_PASSWORD: YWRtaW4xMjM0
+  DB_HOST: postgres
+  DB_USER: root
+  DB_PASSWORD: YWRtaW4xMjM0
```

Comment:
Postgres container password seems to be available in clear text here. 
This is a sensitive data expose according to OWASP 10.
Suggestion: Use an environment variable to pull the passwords from a secure vault.